packages/panels/src/Commands/MakePageCommand.php

```diff
@@ -47,8 +48,36 @@ public function handle(): int
         $resourceClass = null;
         $resourcePage = null;
 
-        $resourceInput = $this->option('resource') ?? text(
-            label: 'What is the resource you would like to create this in?',
+        $panel = $this->option('panel');
+
+        if ($panel) {
+            $panel = Filament::getPanel($panel);
+        }
+
+        if (! $panel) {
+            $panels = Filament::getPanels();
+
+            /** @var Panel $panel */
+            $panel = (count($panels) > 1) ? $panels[select(
+                label: 'Which panel would you like to create this in?',
+                options: array_map(
+                    fn (Panel $panel): string => $panel->getId(),
+                    $panels,
+                ),
+                default: Filament::getDefaultPanel()->getId()
+            )] : Arr::first($panels);
+        }
+
+        $resourceInput = $this->option('resource') ?? suggest(
+            label: 'Which resource would you like to create this in?',
+            options: collect($panel->getResources())
+                ->filter(fn (string $namespace): bool => str($namespace)->contains('\\Resources\\'))
+                ->map(
+                    fn (string $namespace): string => str($namespace)
```

Comment:
```suggestion
                    fn (string $namespace): string => (string) str($namespace)
```

Reply:
Isn't this unnecessary since the return type already casts to string?